Azure.DevOps.Extensions.XrmRelease/Azure.DevOps.Extensions.XrmRelease.Datamigration.PowerShellModule/Cmdlets/DataImporterCmdlet.cs

```diff
@@ -106,6 +106,7 @@ protected override void ProcessRecord()
                 {
                     manager.StartSingleThreadedImport(importConfig, new CmdletLoggerPS(this, TreatWarningsAsErrors), cancellationTokenSource, ConnectionString, CsvImport, schemaConfig);
                 }
+                this.LogTaskCompleteResult(logger);
```

Comment:
This `logger` isn't the same logger that gets passed to `manager.StartSingleThreadedImport()`, so I think this will only work for multi-threaded imports. Probably a little bit of refactoring required.

Reply:
Refactoring done to log for single threaded import as well.